### PR TITLE
Akka 2.6.9 + replicated event sourcing video

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title: Akka 2.6.9 Released
+author: Christopher Batey
+short: We are excited to announce the 2.6.9 patch release of Akka 2.6
+category: news
+tags: [releases]
+---
+
+Dear hakkers,
+
+We are excited to announce a new patch release of Akka 2.6. Notable changes relative to 2.6.6 include:
```
Comment (patriknw) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:12]: relative to 2.6.8

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+* Replicated Event Sourcing a.k.a Multi-DC persistence. See below. [#29217](https://github.com/akka/akka/issues/29217)
+* Chunked messages in reliable delivery. Large messages sent over reliable delivery can now be chunked automatically into smaller ones. [#29560](https://github.com/akka/akka/issues/29560)
+* Embed types in Jackson that have existing Akka serializers. 
+* Improve Shard#entityCreated API performance. [#29520](https://github.com/akka/akka/issues/29217)
```
Comment (patriknw) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:17]: that was a contribution? "thanks to..."

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+2.6.8 includes 81 issues. The complete list can be found on the [2.6.9 milestone](https://github.com/akka/akka/milestone/169?closed=1) on github.
+
+## Replicated Event Sourcing 
+
+Earlier in the year [Lightbend](https://www.lightbend.com/) announced the [open sourcing of two key commercial features](https://www.lightbend.com/blog/lightbend-to-contribute-commercial-features-to-akka-open-source): Split brain resolver and Multi-DC persistence.
+Rather than open sourcing the existing Multi-DC persistence library we have instead integrated the same functionality in to Akka 2.6's event sourcing APIs.
+This is part of a larger effort to unify event sourcing APIs across Akka, Multi-DC persistence, and Lagom.
+In addition, we've added support for running multiple replicas in the same DC and for all databases that have a persistence plugin that supports metadata. 
+Currently, that is Apache Cassandra and Google Spanner.
```
Comment (johanandren) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:32]: "Shortly releases of the Apache Cassandra and Google Spanner plugins will have this support added"?

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+2.6.8 includes 81 issues. The complete list can be found on the [2.6.9 milestone](https://github.com/akka/akka/milestone/169?closed=1) on github.
+
+## Replicated Event Sourcing 
+
+Earlier in the year [Lightbend](https://www.lightbend.com/) announced the [open sourcing of two key commercial features](https://www.lightbend.com/blog/lightbend-to-contribute-commercial-features-to-akka-open-source): Split brain resolver and Multi-DC persistence.
+Rather than open sourcing the existing Multi-DC persistence library we have instead integrated the same functionality in to Akka 2.6's event sourcing APIs.
+This is part of a larger effort to unify event sourcing APIs across Akka, Multi-DC persistence, and Lagom.
+In addition, we've added support for running multiple replicas in the same DC and for all databases that have a persistence plugin that supports metadata. 
+Currently, that is Apache Cassandra and Google Spanner. 
+JDBC has support when not using snapshots and support will be added shortly.
```
Comment (johanandren) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:33]: "and support for that as well"

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+This means that replicated event sourcing supports cross datacenter replication without any special support from the database. Meaning that active-active event sourcing can be
+achieved even when there is no inter-dc replication at the database level e.g. independent relation databases per datacenter.
+
+## Credits
+
+For this release we had the help of ??? committers – thank you all very much!
+
+```
+
+```
```
Comment (johanandren) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:44]: missing

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+* Chunked messages in reliable delivery. Large messages sent over reliable delivery can now be chunked automatically into smaller ones. [#29560](https://github.com/akka/akka/issues/29560)
+* Embed types in Jackson that have existing Akka serializers. 
+* Improve Shard#entityCreated API performance. [#29520](https://github.com/akka/akka/issues/29217)
+
+As well as some important bug fixes:
+
+* EventSourcedBehavior un-stash causes unbounded recursion and JVM StackOverflow exceptions. [#29401](https://github.com/akka/akka/issues/29401)
+* Noisy logs from sharding. [#29400](https://github.com/akka/akka/issues/29400)
+
+2.6.8 includes 81 issues. The complete list can be found on the [2.6.9 milestone](https://github.com/akka/akka/milestone/169?closed=1) on github.
```
Comment (patriknw) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:24]: 2.6.9
don't we normally say "closed issues". I'm sure there are more than 81 bugs remaining :)

Comment (johanandren) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:24]: 2.6.9

File: blog/_posts/2020-09-09-akka-2.6.9-released.md (hunk 1 above)
Comment (johanandren) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:12]: relative to 2.6.8

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+tags: [releases]
+---
+
+Dear hakkers,
+
+We are excited to announce a new patch release of Akka 2.6. Notable changes relative to 2.6.6 include:
+
+* Replicated Event Sourcing a.k.a Multi-DC persistence. See below. [#29217](https://github.com/akka/akka/issues/29217)
+* Chunked messages in reliable delivery. Large messages sent over reliable delivery can now be chunked automatically into smaller ones. [#29560](https://github.com/akka/akka/issues/29560)
+* Embed types in Jackson that have existing Akka serializers.
```
Comment (johanandren) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:16]: no issue/pr link?

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+This means that replicated event sourcing supports cross datacenter replication without any special support from the database. Meaning that active-active event sourcing can be
+achieved even when there is no inter-dc replication at the database level e.g. independent relation databases per datacenter.
+
+## Credits
+
+For this release we had the help of ??? committers – thank you all very much!
+
+```
+
```
Comment (patriknw) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:43]: TODO

File: blog/_posts/2020-09-09-replicated-event-sourcing-video.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+author: Christopher Batey
+short: A short video detailing the new Replicated Event Sourcing feature. 
+category: news
+tags: [videos]
+youtubeId: SuTPY5MxmUM
+---
+
+{% include youtubePlayer.html id=page.youtubeId %}
+
+This video introduces [Replicated Event Sourcing](https://doc.akka.io/docs/akka/snapshot/typed/replicated-eventsourcing.html), new in Akka 2.6.9. Replicated Event Sourcing builds on top of `EventSourcedBehavior`s allowing
```
Comment (patriknw) [inline blog/_posts/2020-09-09-replicated-event-sourcing-video.md:13]: s/snapshot/current/

File: blog/_posts/2020-09-09-akka-2.6.9-released.md (hunk 7 above)
Comment (chbatey) [reply blog/_posts/2020-09-09-akka-2.6.9-released.md:16]: It was hidden in a replicated PR but thought it was worth noting

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 10]
```diff
@@ -40,7 +40,30 @@ achieved even when there is no inter-dc replication at the database level e.g. i
 For this release we had the help of ??? committers – thank you all very much!
 
 ```
-
+commits  added  removed
+     32  15871     1218 Christopher Batey
+     26   3946     1427 Patrik Nordwall
+     24   9394     2578 Johan Andrén
+      7      9        9 Scala Steward
```
Comment (patriknw) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:47]: we could remove bots, but not important

File: blog/_posts/2020-09-09-akka-2.6.9-released.md [hunk 11]
```diff
@@ -40,7 +40,30 @@ achieved even when there is no inter-dc replication at the database level e.g. i
 For this release we had the help of ??? committers – thank you all very much!
```
Comment (patriknw) [inline blog/_posts/2020-09-09-akka-2.6.9-released.md:40]: the count ???

**Review by patriknw — APPROVED** [review]
LGTM, after some minor

**Review by johanandren — APPROVED** [review]

**Pull request merged**